### PR TITLE
Added file checking on execute functions

### PR DESCRIPTION
**Pull request by repletsin5**
Added file checking on execute functions so the application doesn't crash when trying to launch an application that doesn't exist

### CODE REVIEW

File: Shell/Shell.cs [hunk 1]
```diff
@@ -357,20 +357,34 @@ public void Execute(string input, bool waitForExit)
...
+                        process.Start();
 
-                    process.Start();
-                    process.WaitForExit();
+                        if(waitForExit)
+                            process.WaitForExit();
+                    }
+                    else
+                    {
+                        Console.WriteLine("Couldn't find file \"{0}\" to execute. Reinstalling should fix the issue ", Aliases[input]);
```
Comment (0x78654C) [inline Shell/Shell.cs:369]: Use  FileSystem.ErrorWriteLine for output errors.

File: Shell/Shell.cs [hunk 2]
```diff
@@ -357,20 +357,34 @@ public void Execute(string input, bool waitForExit)
...
+                    process.Start();
+                    if (waitForExit)
+                    {
+                        process.WaitForExit();
+                    }
+                }
+                else
                 {
-                    process.WaitForExit();
+                    Console.WriteLine("Couldn't find file \"{0}\" to execute. Reinstalling should fix the issue ", Aliases[input]);
```
Comment (0x78654C) [inline Shell/Shell.cs:387]: Use  FileSystem.ErrorWriteLine for output errors.

File: Shell/Shell.cs [hunk 3]
```diff
@@ -427,12 +448,15 @@ public int ExecuteWithArgs2(string input, string args, string args2, bool waitFo
...
                 {
-                    process.WaitForExit();
-                }
+                    process.Start();
+                    if (waitForExit)
+                    {
+                        process.WaitForExit();
+                    }
+                }else
+                    Console.WriteLine("Couldn't find file \"{0}\" to execute. Reinstalling should fix the issue ", Aliases[input]);
```
Comment (0x78654C) [inline Shell/Shell.cs:459]: Use  FileSystem.ErrorWriteLine for output errors.

**Review by 0x78654C — COMMENTED** [review]
check comments please

File: Shell/Shell.cs [hunk 4]
```diff
@@ -235,7 +235,7 @@ public void Run()
                     if (s_input == "help")
                     {
                         string helpMGS = @"
-xTerminal v1.0 Copyright @ 2020 0x078654c
+xTerminal v1.0 Copyright @ 2020-2021 0x078654c, repletsin5
```
Comment (0x78654C) [inline Shell/Shell.cs:238]: is it normal? license file is to one person created. :D

Comment (repletsin5) [reply Shell/Shell.cs:238]: i added it because i added code to the file

Comment (repletsin5) [reply Shell/Shell.cs:238]: yeh oops i dont know why i added that

Comment (0x78654C) [reply Shell/Shell.cs:238]: is no problem. As far I know only owners (creators ) are on copyright. Contributors are show in left and always mentioned. Beside is help message here:d

**Review by 0x78654C — APPROVED** [review]
LGTM

**Pull request merged**